khal/khalendar/event.py

```diff
@@ -919,110 +933,90 @@ def duration(self) -> dt.timedelta:
 
 
 def create_timezone(
-    tz: pytz.BaseTzInfo,
-    first_date: Optional[dt.datetime]=None,
-    last_date: Optional[dt.datetime]=None
+    tz,
```

Comment:
Please leave type hints here:

```suggestion
    tz: pytz.tzinfo | zoneinfo.ZoneInfo,
```